compiler/resolution/functionResolution.cpp

```diff
@@ -463,6 +463,35 @@ resolveUninsertedCall(Type* type, CallExpr* call) {
   return call->isResolved();
 }
 
+static void resolveFnForCall(FnSymbol* fn, CallExpr* call)
```

Comment:
Style consideration:

I find it disorienting to scan down a file and find functions that have never been used; I wonder what they
are for.  An alternative is to (a) add a function prototype to the top of the file (b) put the definition after
all uses or perhaps the principal use